Makefile

```diff
@@ -17,10 +17,16 @@ uninstall:
 user-%: 
 	$(MAKE) $* prefix="${HOME}" bin=.bin systemd=.config/systemd
 
+#TODO: most likely it is better to get the value from the tomls file
```

Comment:
It would be great if Cargo.toml was the only place that needs a version number, and it would propagate automatically from there. But this is complicated because package versions have an additional number: the package release, the number after the dash.

I think the control file and Cargo.toml each need a hardcoded version number. Keeping the deb filename consistent with the version in Cargo.toml would be ideal, but it doesn't seem as critical as keeping it consistent with the version in control file.